File: ILT.IHR.APP/ClientApp/src/app/core/interfaces/AppraisalGoal.ts
export class IAppraisalGoal {
    AppraisalGoalID: number;
    AppraisalID: number;
    ReviewYear?: number;
    Goal?: string;
    EmpResponse?: number;
    EmpComment?: string;
    MgrResponse?: number;
    MgrComment?: string;
    CreatedBy: string;
    CreatedDate: Date;
    ModifiedBy: string;
    ModifiedDate: Date;
    TimeStamp: ['']

}
export class IAppraisalGoalDisplay {
    appraisalGoalID: number;
    appraisalID: number;
    reviewYear?: number;
    goal?: string;
    empResponse?: number;
    empComment?: string;
    mgrResponse?: number;
    mgrComment?: string;
    createdBy?: string;
    createdDate?: Date;
    modifiedBy?: string;
    modifiedDate?: Date;
    timeStamp?: ['']
}

File: ILT.IHR.DTO/AppraisalGoal.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace ILT.IHR.DTO
{
    public partial class AppraisalGoal : AbstractDataObject
    {
        public int AppraisalGoalID
        {
            get { return base.RecordID; }
            set { base.RecordID = value; }
        }
        public int AppraisalID { get; set; }
        //[Required]
        //[RegularExpression("^0*[1-9]\\d*$")]
        public int ReviewYear { get; set; }
        public string Goal { get; set; }
        public int? EmpResponse { get; set; }
        public string EmpComment { get; set; }
        public int? MgrResponse { get; set; }
        public string MgrComment { get; set; }
   
    }
}
